doc/workflows/anaconda-webui.rst

```diff
@@ -50,12 +50,12 @@ Examples can be found in `default settings file <https://github.com/rhinstaller/
 
 AnacondaWebUI
 ^^^^^^^^^^^^^
-- **anaconda_repo** - URL to Anaconda git repository or path to local directory (use file://)
-- **cockpit_repo** - URL to Cockpit git repository, if anaconda_repo is local directory and
-  ``anaconda/ui/webui/test/common`` exists, this repo won't be used.
+- **webui_repo** - URL to anaconda-webui git repository or path to local directory (use file://)
+- **cockpit_repo** - URL to Cockpit git repository, if webui_repo is local directory and
+  ``./test/common`` exists, this repo won't be used.
 - **cockpit_branch** - Cockpit git branch
 - **bots_repo** - URL to Cockpit Bots git repository, if anaconda_repo is local directory and
```

Comment:
"anaconda_repo" should be replaced with "webui_repo"

Reply:
It is, or am I missing something?